build.assets/Dockerfile-centos7

```diff
@@ -153,19 +193,21 @@ RUN yum groupinstall -y 'Development Tools' && \
     zip \
     # required by libbpf
     zlib-static && \
-    yum -y install https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm && \
-    yum -y install git && \
-    yum clean all
+    yum clean all && \
+    localedef -c -i en_US -f UTF-8 en_US.UTF-8
```

Comment:
What's this for?

Reply:
For some reason, CentOS 7 Docker images come without any locales and generate a bunch of warnings with every command called in the shell. This generates default locales. We are doing this already in out other ARM64 image https://github.com/gravitational/teleport/blob/e8896791adffe427ef602d8fc43d9e9ec08897fb/build.assets/Dockerfile-multiarch#L71